src/wazuh_modules/vulnerability_detector/wm_vuln_detector_evr.c

```diff
@@ -396,14 +396,36 @@ static int rpm_vercmp(const char * a, const char * b)
 *          0: a and b are the same version
 *        < 0: b is newer than a
 */
-static int nvd_verrevcmp(const char *a, const char *b)
+static int nvd_verrevcmp(const char *a, const char *b, int revision)
 {
+    int result;
+    char *rev;
+    size_t rev_size;
+
     // In the NVD version comparison, we avoid the revision
     // comparison if any of them is not present. 
-    if ((a == NULL) || (b == NULL))
+    if ((a == NULL) || (b == NULL)) {
         return 0;
-    else
-        return deb_verrevcmp(a, b);
+    }
+
+    if (revision) {
```

Comment:
Please add a comment explaining why you added this change. And if possible an example.

Reply:
Done! https://github.com/wazuh/wazuh/pull/5035/commits/305b2e8bd08015144cd544545f226fc7ed021436